online_meetings/canvas_zoom_meetings.py

```diff
@@ -0,0 +1,138 @@
+# Script to get all sites where Zoom is visible and retrieve the meetings to generate a report
+
+import json
+import yaml
+import re
+import os
+import sys
+import requests
+import logging
+import http.client
+from datetime import datetime
+
+from canvasapi import Canvas
+from bs4 import BeautifulSoup as bs
+
+import pandas as pd
+
+# read configurations
+try:
+    with open(os.path.join('../config', 'env.json')) as env_file:
+        ENV = yaml.load(env_file.read())
+except FileNotFoundError:
+    sys.exit(
+        'Configuration file could not be found; please add env.json to the config directory.')
+
+LOG_LEVEL = ENV.get('LOG_LEVEL', 'DEBUG')
+logging.basicConfig(level=LOG_LEVEL)
+
+# These two lines enable debugging at httplib level (requests->urllib3->http.client)
+# You will see the REQUEST, including HEADERS and DATA, and RESPONSE with HEADERS but without DATA.
+# The only thing missing will be the response.body which is not logged.
+if LOG_LEVEL == logging.DEBUG:
+    http.client.HTTPConnection.debuglevel = 1
```

Comment:
Can we use a Http logging handler and set the debug level theirs? `basicConfig` logging takes an attribute called `handler` so that we can give as many handlers as we want
https://docs.python.org/3.1/library/logging.html#logging.handlers.HTTPHandler.

We can even have a `logging.yaml ` file to set all the logging configuration etc.

Reply:
I think we can improve the logging on a separate fix, will create an issue. Right now this is still semi-standalone but I think we'll want to have the centralized script do all of the logging setup and be part of that cleanup.